### ISSUE TITLE
Invalid vertex indices

### ISSUE THREAD

**Issue by Aranuvir** [makehumancommunity/community-plugins-makeclothes#8]
This is a bug from the forum: http://www.makehumancommunity.org/forum/viewtopic.php?f=22&t=18540
Under certain (currently unknown) circumstances MC seems to create invalid vertex indices (KeyError: -1). Even if issue might be caused by an initially incorrect setup of the clothes to be generated, MC shouldn't crash here and instead should stop cloth generation and give an appropriate warning.

Here is the traceback:

> Traceback (most recent call last):
> File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\makeclothes\operators\createclothes.py", line 85, in execute
> (b, hint) = mc.make()
> File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\makeclothes\core_makeclothes_functionality.py", line 179, in make
> self.findBestFaces()
> File "C:\Users\Owner\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\makeclothes\core_makeclothes_functionality.py", line 277, in findBestFaces
> for polygon in self.humanmesh.vertPolygons[vertIdx]:
> KeyError: -1
> 
> location: <unknown location>:-1

**Comment by black-punkduck** [comment]
there was a version somewhere in the past, where we did not test the number of assigned vertices of a group on the human mesh. When I added the code for the "rigid groups" I became aware of this problem and since then the program checks for at least three vertices in a group on the human. I tried to figure out the version by comparing line numbers with history, it looks like a version somewhere used in December. Since I did change the code, the user should test if this still happens ...
